### PR TITLE
1613 document kill time

### PR DESCRIPTION
**Pull request by ilija-lazoroski**
# What does this PR do?

Fixes part of #1613.

Add any further explanations here.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] Is the TravisCI build passing?
* [x] Was the documentation framework updated to reflect the changes?

## Testing Checklist

* [x] If applicable, add screenshots or log transcripts of the feature working

image
image

## Explain Changes

Are the commit messages enough? If not, elaborate.

### CODE REVIEW

File: docs/content/FAQ/_index.md [hunk 1]
```diff
@@ -59,6 +60,13 @@ ref "/reference/data_directory" >}}).
 
 The Infection Monkey agent shuts off either when it can't find new victims or it has exceeded the quota of victims as defined in the configuration.
 
+## How long does it take to stop all running Infection Monkey agents?
+
+When <b>Kill All Monkeys</b> is pressed, the agents try to finish execution
+safely. This can take up to 2 minutes, but will be much shorter on average.
+
+![Kill All Monkeys Button](/images/faq/kill_all_monkeys.jpeg "Kill All Monkeys Button")
```
Comment (mssalvatore) [inline docs/content/FAQ/_index.md:68]: Do we need this screenshot? I'm not sure what it really adds.

**Review by mssalvatore — CHANGES_REQUESTED** [review]
Need to rebase on agent-refactor and retarget the PR to agent-refactor.

File: docs/content/FAQ/_index.md (hunk 1 above)
Comment (ilija-lazoroski) [reply docs/content/FAQ/_index.md:68]: I thought just for the users to know about what we are talking about.  Maybe we don't actually need it. Or I can add the modal window screenshot ?

File: monkey/monkey_island/cc/ui/src/components/pages/MapPage.js [hunk 2]
```diff
@@ -103,7 +103,7 @@ class MapPageComponent extends AuthComponent {
             <div className="text-center">Are you sure you want to kill all monkeys?</div>
           </h2>
           <p style={{'fontSize': '1.2em', 'marginBottom': '2em'}}>
-            This might take a few moments...
+            This might take up to two minutes...
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/ui/src/components/pages/MapPage.js:106]: Maybe a number and in bold?

File: docs/content/FAQ/_index.md [hunk 3]
```diff
@@ -59,6 +60,11 @@ ref "/reference/data_directory" >}}).
 
 The Infection Monkey agent shuts off either when it can't find new victims or it has exceeded the quota of victims as defined in the configuration.
 
+## How long does it take to stop all running Infection Monkey agents?
+
+When <b>Kill All Monkeys</b> is pressed, the agents try to finish execution
```
Comment (shreyamalviya) [inline docs/content/FAQ/_index.md:65]:
```suggestion
On the Infection Map page, when <b>Kill All Monkeys</b> is pressed, the agents try to finish execution
```
or something similar indicating where the button is located

**Pull request merged**